pyomo/contrib/pyros/pyros_algorithm_methods.py

```diff
@@ -75,20 +219,23 @@ def ROSolver_iterative_solve(model_data, config):
             config=config,
         )
         if not coeff_matching_success and not robust_infeasible:
-            raise ValueError(
-                "Equality constraint \"%s\" cannot be guaranteed to be robustly feasible, "
-                "given the current partitioning between first-stage, second-stage and state variables. "
-                "You might consider editing this constraint to reference some second-stage "
-                "and/or state variable(s)." % c.name
+            config.progress_logger.error(
+                f"Equality constraint {c.name!r} cannot be guaranteed to "
+                "be robustly feasible, given the current partitioning "
+                "between first-stage, second-stage, and state variables. "
```

Comment:
Grammar (minor nit), but 'between' is for comparing 2 things.  'among' if for 3 or more.